src/main/resources/avro/metadata.avdl

```diff
@@ -122,10 +122,25 @@ For data accessors, they're a simple way to scope exploration and analysis
 what's the distribution of that result in the data from our project?)   
 */
 record Dataset {
-  /** The dataset UUID. This is globally unique. */
+  /**
+  The dataset's id. This is the local/legacy/generic id.
+  */
   string id;
 
-  /** A description of the dataset. */
+  /**
+  The dataset's guid. This is globally unique id (UUIDv4).
+  TODO: Required?!
+  */
```

Comment:
@dglazer @diekhans Please see https://github.com/ga4gh/schemas/pull/397 (and the metadata branch).